workers/data_refinery_workers/processors/salmon_quant.yml

```diff
@@ -11,6 +11,7 @@ os_pkg:
 # Command lines (commands whose outputs will be kept track of)
 cmd_line:
   - salmon --version
+  - fasterq-dump --version
```

Comment:
We no longer use fasterq-dump, I guess this sholuld be fastq-dump?

I also suggest that sra-stat should go in here